### ISSUE TITLE
Upgrade to Libsass 3.4

### ISSUE THREAD

**Issue by sphinxc0re** [sass/sassc-ruby#58]
The current version of Libsass doesn't seem to compile this correctly:
```sass
.nav
  border-radius: 2px
  &-
    &factory
      text: bold
    &figure
      border-radius: 2px
      &__
        &hallo
          color: black
          &--
            &red
              color: red
```
although it's valid sass code. So I would suggest upgrading to Libsass 3.4

**Comment by sphinxc0re** [comment]
This will be fixed by #57

**Comment by bolandrm** [comment]
Done, thanks
